### ISSUE TITLE
Pretrained model

### ISSUE THREAD

**Issue by MSunDYY** [Haiyang-W/DSVT#79]
Dear Wang, I want to reproduce your work, but the training on waymo is very time-costing. Could you provice me the DSVT-4f model trained on waymo-dataset??? If so, please sent it to my email: 2563733583@qq.com. Looking forward to your reply!

**Comment by Haiyang-W** [comment]
We may not be able to provide the multi-frame weights due to how long ago they were, but I can directly share the single-frame ones with you.

I'm very sorry that I cannot provide you with the multi-frame model, because they are lost.

**Comment by avi9700** [comment]
> We may not be able to provide the multi-frame weights due to how long ago they were, but I can directly share the single-frame ones with you.
> 
> I'm very sorry that I cannot provide you with the multi-frame model, because they are lost.

Dear Wang, 
I wanted to study your model, so I trained it using the Waymo Open Dataset (v1.3.1) and 4× A6000 GPUs.
When I trained with a learning rate of 0.003, the loss became NaN for some reason, so I lowered the learning rate to 0.002.
(USE_SHARED_MEMORY = False)
Although the training completed, the performance did not meet expectations.
I want to try again, but I currently don't have the resources to retrain the model.
If possible, I would greatly appreciate it if you could share a pretrained model for dsvt_plain_1f_onestage.
My email is  seo.niceman@gmail.com .
Thank you very much for your time and support.



OBJECT_TYPE_TYPE_VEHICLE_LEVEL_1/AP: 0.6193 
OBJECT_TYPE_TYPE_VEHICLE_LEVEL_1/APH: 0.6149 
OBJECT_TYPE_TYPE_VEHICLE_LEVEL_1/APL: 0.6193 
OBJECT_TYPE_TYPE_VEHICLE_LEVEL_2/AP: 0.5407 
OBJECT_TYPE_TYPE_VEHICLE_LEVEL_2/APH: 0.5368 
OBJECT_TYPE_TYPE_VEHICLE_LEVEL_2/APL: 0.5407 
OBJECT_TYPE_TYPE_PEDESTRIAN_LEVEL_1/AP: 0.7591 
OBJECT_TYPE_TYPE_PEDESTRIAN_LEVEL_1/APH: 0.6781 
OBJECT_TYPE_TYPE_PEDESTRIAN_LEVEL_1/APL: 0.7591 
OBJECT_TYPE_TYPE_PEDESTRIAN_LEVEL_2/AP: 0.6781 
OBJECT_TYPE_TYPE_PEDESTRIAN_LEVEL_2/APH: 0.6041 
OBJECT_TYPE_TYPE_PEDESTRIAN_LEVEL_2/APL: 0.6781 
OBJECT_TYPE_TYPE_SIGN_LEVEL_1/AP: 0.0000 
OBJECT_TYPE_TYPE_SIGN_LEVEL_1/APH: 0.0000 
OBJECT_TYPE_TYPE_SIGN_LEVEL_1/APL: 0.0000 
OBJECT_TYPE_TYPE_SIGN_LEVEL_2/AP: 0.0000 
OBJECT_TYPE_TYPE_SIGN_LEVEL_2/APH: 0.0000 
OBJECT_TYPE_TYPE_SIGN_LEVEL_2/APL: 0.0000 
OBJECT_TYPE_TYPE_CYCLIST_LEVEL_1/AP: 0.7030 
OBJECT_TYPE_TYPE_CYCLIST_LEVEL_1/APH: 0.6905 
OBJECT_TYPE_TYPE_CYCLIST_LEVEL_1/APL: 0.7030 
OBJECT_TYPE_TYPE_CYCLIST_LEVEL_2/AP: 0.6775 
OBJECT_TYPE_TYPE_CYCLIST_LEVEL_2/APH: 0.6654 
OBJECT_TYPE_TYPE_CYCLIST_LEVEL_2/APL: 0.6775 

2025-05-27 11:15:44,506   INFO  Result is save to /docker_ws/src/output/cfgs/dsvt_models/dsvt_plain_1f_onestage/default/eval/epoch_1/val/default
2025-05-27 11:15:44,506   INFO  ****************Evaluation done.*****************
